src/components/BookingCalendar/container.js

```diff
@@ -70,35 +70,35 @@ const BookingCalendarContainer = Wrapped =>
       }
     };
 
-    onSelectSlot = ({ start, end }) => {
-      const calendarValidationResults = this.handleCalendarValidation(
-        start,
-        end,
-      );
-      if (calendarValidationResults === validationMessage.DATES_APPROVED) {
-        let booking = {
-          start: new moment(start),
-          end: new moment(end),
-          isHalfday: false,
-          eventType: {
-            eventTypeId: eventTypes.ANNUAL_LEAVE,
-            description: 'Annual leave',
-          },
-          eventStatus: {
-            eventStatusId: holidayStatus.PENDING,
-            description: 'Awaiting Approval',
-          },
-          employee: null,
-        };
-        this.props.selectBooking(booking);
-        this.bookingModalConfig({ ...booking }, false);
-      } else {
-        Toast({
-          type: 'warning',
-          title: calendarValidationResults,
-        });
-      }
-    };
+    // onSelectSlot = ({ start, end }) => {
```

Comment:
Is this going to be coming back in at another point? If not could it be deleted?

Reply:
It has been a while since I was working on this so I could be wrong, but I believe I had commented this out because it was causing problems with the updated back end. It needs a refactor pending the new dashboard API from the back-end. This may be something I look at today once I find out the progress on it.